### PR TITLE
Supporting both flags and envs. Updated Dockerfile

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

**Comment by tamas-molnar** [comment]
lgtm

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,21 @@
-FROM alpine
+FROM alpine:3.3
 
-ENV GOPATH /gopath
-ENV REPO_PATH "github.com/Financial-Times/subjects-transformer"
+ADD *.go /subjects-transformer/
+ADD handlers/*.go /subjects-transformer/handlers/
+ADD service/* /subjects-transformer/service/
+ADD model/*.go /subjects-transformer/model/
```
Comment (george-angel) [inline Dockerfile]: Can we just `ADD .` to reduce layers?

Comment (catalin-ilea) [inline Dockerfile]: Done

**Comment by george-angel** [comment]
+1

**Pull request merged**